bin/generateAcronyms.py

```diff
@@ -883,9 +893,16 @@ def dump_gls(filename, out_file):
         help=""" Generate glossary dump file using passed
                                  filename containing  all entries.""",
     )
+    parser.add_argument(
+        "-s",
+        "--skipnone",
+        action="store_true",
+        help=""" Do not load skip acronyms file""",
```

Comment:
```suggestion
        help="""Do not load skip acronyms file""",
```
"skipnone" meaning "skip the skip acronyms file" is a bit tortured. How about "--skip-acronyms/--no-skip-acronyms" defaulting to true and have the variable be "skip_acronyms". That would then allow the code to say "if skip_acronyms" which would be easier to understand.